__tests__/propertyVolumeTests.ts

```diff
@@ -3,13 +3,13 @@ process.env['NODE_CONFIG_DIR'] = __dirname + '/config/'
 import * as properties from '../src'
 import * as config from 'config'
 
-properties.addTo(config, '__tests__/testVolumes/testVol/')
-properties.addTo(config, '__tests__/testVolumes/secrets')
+properties.addTo(config, {mountPoint: '__tests__/testVolumes/testVol/'})
+properties.addTo(config, {mountPoint: '__tests__/testVolumes/secrets'})
```

Comment:
```suggestion
properties.addTo(config, { mountPoint: '__tests__/testVolumes/secrets' })
```

Reply:
same here do we have a standard lint settings since I had to change webstorm from this to the lint file settings.